tools/deployment/package/docker-compose.yml

```diff
@@ -0,0 +1,388 @@
+    environment:
+      PYTHONPATH: "/opt/clp/lib/python3/site-packages"
+    depends_on:
+      results-cache:
+        condition: "service_healthy"
+    command: [
+      "python3",
+      "-u",
+      "-m", "clp_py_utils.initialize-results-cache",
+      "--uri", "mongodb://results_cache:27017/${RESULTS_CACHE_CLP_DB_NAME:-clp-query-results}",
+      "--stream-collection", "${CLP_RESULTS_CACHE_STREAM_COLLECTION_NAME:-stream-files}",
+    ]
+
+  compression-scheduler:
+    <<: *service_defaults
+    container_name: "compression_scheduler"
+    image: "${CLP_PACKAGE_CONTAINER}"
+    environment:
+      BROKER_URL: "amqp://${CLP_QUEUE_USER}:${CLP_QUEUE_PASS}@queue:5672"
+      CLP_DB_PASS: "${CLP_DB_PASS}"
+      CLP_DB_USER: "${CLP_DB_USER}"
+      CLP_LOGGING_LEVEL: "${CLP_COMPRESSION_SCHEDULER_LOGGING_LEVEL:-INFO}"
+      CLP_LOGS_DIR: "/var/log"
+      PYTHONPATH: "/opt/clp/lib/python3/site-packages"
+      RESULT_BACKEND: >-
+        redis://default:${CLP_REDIS_PASS}@redis:6379/${CLP_REDIS_COMPRESSION_BACKEND_DB:-1}
+    volumes:
+      - "${CLP_AWS_CONFIG_DIR_HOST:-~/.aws}:/.aws:ro"
+      - "${CLP_COMPRESSION_SCHEDULER_LOGS_FILE_HOST:-./var/log/compression_scheduler.log}\
+:/var/log/compression_scheduler.log"
+      - "${CLP_LOGS_DIR_HOST:-./var/log}/.clp-config.yml:/etc/clp-config.yml:ro"
+      - "/:/mnt/logs:ro"
+    depends_on:
+      db-table-creator:
+        condition: "service_completed_successfully"
+      queue:
+        condition: "service_healthy"
+      redis:
+        condition: "service_healthy"
+    command: [
+      "python3",
+      "-u",
+      "-m", "job_orchestration.scheduler.compress.compression_scheduler",
+      "--config", "/etc/clp-config.yml"
+    ]
+
+  query-scheduler:
+    <<: *service_defaults
+    container_name: "query_scheduler"
+    image: "${CLP_PACKAGE_CONTAINER}"
+    environment:
+      BROKER_URL: "amqp://${CLP_QUEUE_USER}:${CLP_QUEUE_PASS}@queue:5672"
+      CLP_DB_PASS: "${CLP_DB_PASS}"
+      CLP_DB_USER: "${CLP_DB_USER}"
+      CLP_LOGGING_LEVEL: "${CLP_QUERY_SCHEDULER_LOGGING_LEVEL:-INFO}"
+      CLP_LOGS_DIR: "/var/log"
+      PYTHONPATH: "/opt/clp/lib/python3/site-packages"
+      RESULT_BACKEND: >-
+        redis://default:${CLP_REDIS_PASS}@redis:6379/${CLP_REDIS_QUERY_BACKEND_DB:-0}
+    volumes:
+      - "${CLP_LOGS_DIR_HOST:-./var/log}/.clp-config.yml:/etc/clp-config.yml:ro"
+      - "${CLP_QUERY_SCHEDULER_LOGS_FILE_HOST:-./var/log/query_scheduler.log}:\
+/var/log/query_scheduler.log"
+    depends_on:
+      db-table-creator:
+        condition: "service_completed_successfully"
+      queue:
+        condition: "service_healthy"
+      redis:
+        condition: "service_healthy"
+    command: [
+      "python3",
+      "-u",
+      "-m", "job_orchestration.scheduler.query.query_scheduler",
+      "--config", "/etc/clp-config.yml"
+    ]
+    healthcheck:
+      <<: *healthcheck_defaults
+      # FIXME: need to suppressing warnings in the scheduler for reading 0 out of 8 expected bytes
```

Comment:
if we agree that this is the best way to check liveness of the query scheduler, i'll submit a separate PR to gracefully handle such requests in the query scheduler